torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -542,8 +542,10 @@ Stmt* LoopNest::lowerToStmt(Tensor* t) {
 
 class FunctionInliner : public IRMutator {
  public:
-  FunctionInliner(Store* producer)
-      : buf_(producer->buf()), producer_(producer) {
+  FunctionInliner(
+      Store* producer,
+      const std::unordered_set<const Buf*>& outputs)
```

Comment:
Minor: I mildly prefer a LoopNest here, and then call a OutputBufs() internally. That way, if we need more info later, which is Class is entitled to, we don't have to change the signature very often. It is up to you.

Reply:
Let's do this once the need for the extra info emerges.